### ISSUE TITLE
Favorite plugins page

### ISSUE THREAD

**Issue by ganesshkumar** [ganesshkumar/obsidian-plugins-stats-ui#6]

**Comment by ooker777** [comment]
How does the site remember my favorite plugins? Via cache? I'm not sure if registering account is ok for more permanent storage?

**Comment by ganesshkumar** [comment]
Yes, I am using the localStorage of the browser to store user's favorite list and it doesn't sync across browsers or devices. 

As of now, the site is a light weight statically generated site. Not sure if we should add login capabilities and expose a backend just for favorites use case. 

One alternative could be allowing the user to export the list of favorite plugins via json file and import it on another device.

**Comment by ooker777** [comment]
yeah, I think we shouldn't make the site heavier. I have a better idea: why don't we make a plugin to store this data, or to combine with BRAT?

**Comment by ganesshkumar** [comment]
> why don't we make a plugin to store this data

Mobile/tablets, PWA etc. won't have the support for plugins. Also, what will plugin provide us? Will it let us sync the data across devices? I don't think it is possible without supporting authentication on the plugin end.

  > or to combine with BRAT?
  
  I know BRAT as a Beta plugins place. Are you suggesting to include the plugins from BRAT or store data like how BRAT does? (I haven't explored this yet, tbh)

**Comment by djasonic** [comment]
A feature enabling user to load their local vault .obsidian/community-plugins.json file up to the page import the local session favorites of the site would be very convenient.  

Then it becomes reasonably possible for users to sync site favorites to match the installed set of plugins of their vault synced on any device, and still avoid overloading the site with authentication for it.
